### PR TITLE
[frontend] Fix vulnerabilities reported by npm audit

### PR DESCRIPTION
**Pull request by JohanAhlen**
The following packages were automatically upgraded to safe versions by running "npm audit fix":

adobe/css-tools  <4.3.1
Severity: moderate
Regular Expression Denial of Service (ReDOS) while Parsing CSS - https://github.com/advisories/GHSA-hpx4-r86g-5jrg

@babel/traverse  <7.23.2
Severity: critical
Babel vulnerable to arbitrary code execution when compiling specifically crafted malicious code - https://github.com/advisories/GHSA-67hx-6x53-jw92

json5  <1.0.2
Severity: high
Prototype Pollution in JSON5 via Parse Method - https://github.com/advisories/GHSA-9c47-m6qq-7p4h

postcss  <8.4.31
Severity: moderate
PostCSS line return parsing error - https://github.com/advisories/GHSA-7fh5-64p2-3v2j

semver  <5.7.2 || >=7.0.0 <7.5.2
Severity: moderate
semver vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-c2qf-rxjj-qqgw

Manually tested

### CODE REVIEW

**Review by agl29 — APPROVED** [review]
LGTM!

File: package-lock.json [hunk 1]
```diff
@@ -142,15 +142,16 @@
...
+      "version": "4.3.1",
+      "resolved": "https://registry.npmjs.org/@adobe/css-tools/-/css-tools-4.3.1.tgz",
+      "integrity": "sha512-/62yikz7NLScCGAAST5SHdnjaDJQBDq0M2muyRTpf2VQhw6StBg2ALiu73zSJQ4fMVLA+0uBhBHAle7Wg+2kSg==",
       "dev": true
     },
     "node_modules/@ampproject/remapping": {
       "version": "2.2.1",
       "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.1.tgz",
       "integrity": "sha512-lFMjJTrFL3j7L9yBxwYfCq2k6qqwHyzuUl/XBnif78PWTJYyL/dfowQHWE3sp6U6ZzqWiiIZnpTMO96zhkjwtg==",
+      "dev": true,
```
Comment (Harshg999) [inline package-lock.json:154]: Curious: Any reason why we are adding `"dev": true,` for almost all packages now?

Comment (JohanAhlen) [reply package-lock.json:154]: Not sure, this file is generated automatically.

Comment (bjornalm) [reply package-lock.json:154]: Basically it means it not used in production:
If true then this dependency is either a development dependency ONLY of the top level module or a transitive dependency of one. This is false for dependencies that are both a development dependency of the top level and a transitive dependency of a non-development dependency of the top level.

**Review by bjornalm — APPROVED** [review]
Nice work

**Pull request merged**